types/bluebird/bluebird-tests.ts

```diff
@@ -839,9 +839,13 @@ obj = Promise.promisifyAll(obj, {
 
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-//TODO enable generator
+const generator = function* (a: number, b: string) {return "string"}
+const coroutine = Promise.coroutine<number, string, string>(generator);
+coroutine(5, "foo").then((x: string) => {});
+
+const coroutineCustomYield = Promise.coroutine(generator, { yieldHandler: (value) => "whatever" })
 /*
- func = Promise.coroutine(f);
+ //TODO enable generator
```

Comment:
I personally forgot what was meant with `enable generator`, so IMO you can remove that comment since you rewrote the generator test anyways.